static/src/javascripts/projects/commercial/modules/third-party-tags/audience-science-pql.js

```diff
@@ -28,36 +28,52 @@ define([
     }
 
     function onLoad() {
-        var asiPlacements = window.asiPlacements;
-        var segments = storage.local.get(storageKey) || {};
-        // override the global value with our previously stored one
-        window.asiPlacements = segments[section];
-        segments[section] = asiPlacements;
-        storage.local.set(storageKey, segments);
+        isLoaded = true;
+        mediator.emit('commercial:audience-science:loaded');
```

Comment:
We can't assume DFP will be loaded. But you're right, even if it is not loaded the googletag queue is already present. Good catch!